synse/http_integration_test.go

```diff
@@ -570,20 +581,20 @@ func TestIntegration_TagsOptions(t *testing.T) {
 			},
 		},
 		{
-			"single tag, multiple matches",
+			"single tag group, invalid tag, no match",
```

Comment:
I might change this name just for clarity, particularly the "invalid tag". `bar/foo` is a valid tag, there is just no match for it.